### PR TITLE
Add template code for missing Lexers (issue #1258)

### PR DESCRIPTION
**Pull request by Cyrillev91**
See my coment in Issue #1258

### CODE REVIEW

File: PowerEditor/src/Parameters.h [hunk 1]
```diff
@@ -99,6 +99,10 @@ const int LANG_INDEX_TYPE2 = 3;
 const int LANG_INDEX_TYPE3 = 4;
 const int LANG_INDEX_TYPE4 = 5;
 const int LANG_INDEX_TYPE5 = 6;
+const int LANG_INDEX_TYPE6 = 7;
+const int LANG_INDEX_TYPE7 = 8;
+const int LANG_INDEX_TYPE8 = 9; /* Sintillia : keyword list 0 to 8 (KEYWORDSET_MAX = 8)*/
```
Comment (MAPJe71) [inline PowerEditor/src/Parameters.h]: Typo: Scintilla

File: PowerEditor/src/ScitillaComponent/ScintillaEditView.cpp [hunk 2]
```diff
@@ -1156,6 +1229,23 @@ void ScintillaEditView::setLexer(int lexerID, LangType langType, int whichList)
...
+	{
+		const char * keyWords_char = wmc->wchar2char(pKwArray[LANG_INDEX_TYPE6], CP_ACP);
+		setKeywords(langType, keyWords_char, LANG_INDEX_TYPE6);
+	}
+	if (whichList & LIST_8)
+	{
+		const char * keyWords_char = wmc->wchar2char(pKwArray[LANG_INDEX_TYPE7], CP_ACP);
+		setKeywords(langType, keyWords_char, LANG_INDEX_TYPE7);
+	}
+	//Sintillia : keyword list 0 to 8 (KEYWORDSET_MAX = 8)
```
Comment (MAPJe71) [inline PowerEditor/src/ScitillaComponent/ScintillaEditView.cpp]: Typo: Scintilla

**Comment by Croydon** [comment]
Thanks!

**Comment by Cyrillev91** [comment]
@MAPJe71 PR updated (Scintilla)

**Comment by donho** [comment]
No empty template. Please add only lexer you need.

**Comment by Cyrillev91** [comment]
Hi @donho and all,
Ok, I would suggest again a PR when I should add multiple Lexer.

But as Mr. @dail8859 says in issue #1258 "Unfortunately adding a new lexer is a bit tedious, and every time a new lexer is added all other PRs for lexers need fixed because there will be merge conficts :("
"Usually about 10 files need modified. For example if you take a look here you can see this pending PR for adding Visual Prolog."

This PR was to simplify adding lexer and "connect" lexer already in Scintilla.

Until I create a new PR, can you add the following PR ? :
- #1257 Adding Scintilla Lexer for motorola S-REC file, Intel Hex, and TEHEX
- #1439 Adding Scintilla Lexer for VISUALPROLOG

**Pull request closed without merging**